### DISCUSSION TITLE
"Block for this client only" vs "Disallow client"

### DISCUSSION THREAD

**Discussion by cu** [AdguardTeam/AdGuardHome#6763] [Q&A]
Hello,

When looking at the query log, there is a 3-dots menu to the right of each entry, it says:

* Block
* Block for this client only
* Disallow this client

I think I understand what "Block" means (add the domain to a local query list that applies to all clients), but what is the difference between "Block" and "Disallow" in this context?

**Comment by whyisthisbroken** [comment]
Disallowed Clients means, AdGuard Home will drop requests from these clients. All of them...

You will find the list in "settings\dns settings"
